docker/nginx/default.conf

```diff
@@ -0,0 +1,10 @@
+server {
+    server_name search--dev;
```

Comment:
Will nginx be used for dev purposes?

Reply:
No, in a `dev` environment (a person with a laptop) the Flask standalone server should be enough to run code and debug it. What really is important is to have in Continuous Integration an environment that is production-like, so eventually we will have in the build some smoke tests that try to load `/ping` or other APIs through the nginx and WSGI app server combination.

Reply:
Can these tests be done on the actual production search container before deployment?

Reply:
Conflating image and container here, but usually we do:
- run smoke tests on the _image_ when put into a production-like environment like `ci` or `end2end` or anything else that is available for testing. This finds problem in the image itself.
- run smoke tests on the actual _container_ in the real production environment, _after_ deployment (because you need to start the container with the new image to be able to do it). This finds problem in the configuration being used.
So the answer is: yes, but we don't have those tests yet. I'll add them to this PR and cover the first of these two points.